engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -24,23 +24,25 @@
 import org.terasology.rendering.opengl.FBO;
 import org.terasology.rendering.opengl.FBOConfig;
 import org.terasology.rendering.opengl.ScreenGrabber;
+import org.terasology.rendering.opengl.SwappableFBO;
 
 /**
  * TODO: Add javadocs
  * TODO: Better naming
  */
 public class DisplayResolutionDependentFBOs extends AbstractFBOsManager {
     // TODO: Shift this kind of swapping functionality to a more generic SwappableFBO class
-    public static final ResourceUrn BUFFER1 = new ResourceUrn("engine:sceneOpaque1");
-    public static final ResourceUrn BUFFER2 = new ResourceUrn("engine:sceneOpaque2");
-    public static final ResourceUrn FINAL_BUFFER = new ResourceUrn("engine:sceneFinal");
+    public static final ResourceUrn GBUFFER = new ResourceUrn("engine:gBuffer");
+    public static final ResourceUrn GBUFFER_READ = new ResourceUrn("engine:gBuffer");
+    public static final ResourceUrn GBUFFER_WRITE = new ResourceUrn("engine:gBuffer");
+    public static final ResourceUrn FINAL_BUFFER = new ResourceUrn("engine:final");
```

Comment:
And I also think the variable names should be: READONLY_GBUFFER and WRITEONLY_GBUFFER.